sass/site/primary/_posts-and-pages.scss

```diff
@@ -2,6 +2,18 @@
 	display: block;
 }
 
+.sticky-post {
```

Comment:
This looks good, thanks @LittleBigThing! My only suggestion would be to change this `sticky-post` class name to something that's slightly more descriptive: `sticky-label` or `sticky-post-label` for instance. (At first read, I wrongly assumed `sticky-post` was a wrapper for the entire post).

Reply:
Did you know that titles can contain markup? So if it has some word in bold, it won't be any different.

Reply:
> Did you know that titles can contain markup? So if it has some word in bold, it won't be any different.

To make  sure I'm clear: you're referring to the post title? Yes, that can contain limited markup. However, I don't believe that is related to this specific PR. The only element we're making bold here is the "FEATURED" label above the post title. Since we know exactly what that says, I don't think it needs any special considerations. 

As an aside though, we actually do have styles in place to make sure that bold post titles are displayed in a bolder weight:

<img width="475" alt="screen shot 2018-10-19 at 10 44 28 pm" src="https://user-images.githubusercontent.com/1202812/47250545-d9884b80-d3f0-11e8-8aa7-4b4d3a14d943.png">

Reply:
Cool. I couldn't tell from your comment which part should be bold.